members/SMDL0a3a0cdf.yaml

```diff
@@ -0,0 +1,54 @@
+bioguide: SMDL0a3a0cdf
+contact_form:
+  method: post
+  action: ""
+  steps:
+    - visit: "http://mgaleg.maryland.gov/webmga/frmMain.aspx?pid=sponpage&tab=subject6&id=crosby01&stab=01&ys=2019RS&poptype=contact&popid=crosby01"
```

Comment:
Is there any way to remove any non-required url query strings from the form url?

If all these url query strings are required, never mind!

Reply:
I don't think so, but also not entirely sure. I tried different variations of the URL and couldn't get it to work. The query strings present seem to be necessary. Happy to try other ways to make these URLs shorter if you have any suggestions

Reply:
Looks like you can remove the year, so the updated url would be `http://mgaleg.maryland.gov/webmga/frmMain.aspx?pid=sponpage&tab=subject6&id=crosby01&stab=01&poptype=contact&popid=crosby01`.

I was thinking there's a chance the urls will be invalid after this legislative session even if the member is still in office, and there's a chance you'd have to go back and update them all at that time.

Reply:
oh wow yeah. That actually makes sense. And good point RE legislative sessions. I might be worth just changing those URLs now rather than later. Give a few to do that now. Thanks for flagging!